tools/eslint-config/rules/typescript.js

```diff
@@ -5,6 +5,8 @@ module.exports = {
     '@typescript-eslint/indent': ['error', 2],
     '@typescript-eslint/interface-name-prefix': 'off',
     '@typescript-eslint/explicit-member-accessibility': ['warn'],
+    '@typescript-eslint/prefer-optional-chain': ['warn'],
+    // '@typescript-eslint/prefer-nullish-coalescing': ['warn'], // FIXME define parserOptions.project to generate parserServices
```

Comment:
is it still WIP ?

Reply:
That is a problem that involves more rules behind. I'll detail in an specific issue and try to create a way to face it